.wp-env.json

```diff
@@ -0,0 +1,18 @@
+{
+  "core": null,
+  "plugins": [
+    ".",
+    "./tests/Basic-Auth-master"
+  ],
+  "config": {
+    "WP_ALLOW_MULTISITE": true
+  },
```

Comment:
We need to remove this line for the `multisite-convert` command to work. Otherwise, we will get this error:

```
> wp-env run tests-cli "wp core multisite-convert --title='Distributor Multisite'"

ℹ Starting 'wp core multisite-convert --title='Distributor Multisite'' on the tests-cli container. 

[+] Running 2/0
 ⠿ Container c6d05ada51b8eda479c4ae247b278e53-tests-mysql-1      Running                                   0.0s
 ⠿ Container c6d05ada51b8eda479c4ae247b278e53-tests-wordpress-1  Running                                   0.0s
The network already exists.
✔ Ran `wp core multisite-convert --title='Distributor Multisite'` in 'tests-cli'. (in 1s 42ms)
```